test/contact-summary/contact-summary-emitter.spec.js

```diff
@@ -29,6 +29,17 @@ describe('contact-summary-emitter', function() {
       expect(appliesIf.args[0]).to.deep.eq([undefined]);
       expect(appliesIf.args[1]).to.deep.eq([report]);
     });
+
+    it('allows appliesToType to be an array', () => {
```

Comment:
should we test 
* the case where the appliesTo has an array which doesn't match the form
* the case where the appliesToType has no value
* appliesToType is an empty array
* the positive case for the negative syntax (`['!report']`)
* the negative case for the negative syntax 
* how does this behave for docs with undefined types?